src/Symfony/Component/Form/Extension/Core/Type/CountryType.php

```diff
@@ -42,13 +42,15 @@ public function configureOptions(OptionsResolver $resolver)
         $resolver->setDefaults([
             'choice_loader' => function (Options $options) {
                 $choiceTranslationLocale = $options['choice_translation_locale'];
+                $alpha3 = $options['alpha3'];
 
-                return new IntlCallbackChoiceLoader(function () use ($choiceTranslationLocale) {
-                    return array_flip(Countries::getNames($choiceTranslationLocale));
+                return new IntlCallbackChoiceLoader(function () use ($choiceTranslationLocale, $alpha3) {
+                    return array_flip($alpha3 ? Countries::getAlpha3Names($choiceTranslationLocale) : Countries::getNames($choiceTranslationLocale));
                 });
             },
             'choice_translation_domain' => false,
             'choice_translation_locale' => null,
+            'alpha3' => false,
```

Comment:
you should set the allowed types (bool) below as well

Reply:
Good idea.. i've added it..